app/src/main/java/com/example/sharingang/MapFragment.kt

```diff
@@ -61,25 +52,20 @@ class MapActivity : AppCompatActivity(), OnMapReadyCallback {
         }
         binding.mapView.onCreate(savedInstanceState)
         binding.mapView.getMapAsync(this)
+        return binding.root
     }
 
-
+    @SuppressLint("MissingPermission")
```

Comment:
I don't quite remember, but didn't you say that there was a way to get rid of this? I think @betrisey mentionned it last week.

Reply:
What I mentioned was for this lint error that was a false positive and will be fixed with the next release of the AndroidX activity library
https://github.com/Sharingang/Sharingang-Android/blob/26aa588d0865a3d614c5fe04b1a826d02048c243/app/build.gradle#L47

Reply:
I think it was @SuppressLint("InvalidFragmentVersionForActivityResult"). Missing permission is warning that appears in the code when the permission is not checked right before the feature that requires the permission is called. Since I tried to refactor to have the permission check and the usage of the permission in two separate locations (to avoid having "copy-pasted" code), I had to use this @SuppressLint("MissingPermission")

Reply:
Oh yeah I thought it was already fixed, didn't remember it would be in the next release, my bad.